cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/SqlProviderCache.kt

```diff
@@ -166,27 +164,12 @@ class SqlProviderCache(private val backingStore: WriteableCache) : ProviderCache
     try {
       MDC.put("agentClass", "$source putCacheResult")
 
-      // TODO every source type should have an authoritative agent and every agent should be authoritative for something
-      // TODO terrible hack because no AWS agent is authoritative for clusters, fix in ClusterCachingAgent
-      // TODO same with namedImages - fix in AWS ImageCachingAgent
-      if (
-        source.contains("clustercaching", ignoreCase = true) &&
-        !authoritativeTypes.contains(CLUSTERS.ns) &&
-        cacheResult.cacheResults
-          .any {
-            it.key.startsWith(CLUSTERS.ns)
-          }
-      ) {
-        authoritativeTypes.add(CLUSTERS.ns)
-      } else if (
-        source.contains("imagecaching", ignoreCase = true) &&
-        cacheResult.cacheResults
-          .any {
-            it.key.startsWith(NAMED_IMAGES.ns)
-          }
-      ) {
-        authoritativeTypes.add(NAMED_IMAGES.ns)
-      }
+      // TODO terrible hack because some types are global and a single agent can't be authoritative
```

Comment:
Do you have some weird ide code formatting ... seems like your first line `//` is whitespace differently than what follows.

Reply:
I think it is an idea default but I can fix

Reply:
this is an intellij feature where a TODO includes all the indented lines

I've just marked this as not a TODO and rather a (normally formatted) comment about why it is here

Reply:
I did end up leaving a multi-line (funky indented) TODO in there but separated out a bunch of the comment into a more conventional format